Docs/reference/content/reference/driver/ssl.md

```diff
@@ -72,9 +72,12 @@ Prior to v2.7.0, the driver enabled certificate revocation checking by default,
 |         | .NET Core 2.1         | Yes    | Yes    | Yes | No                |
 
 
-#### Notes:
- - SNI (Server Name Indication) required for Atlas free tier.
- - If a server's certificate includes Certificate Revocation List (CRL) Distribution Points but does not include an Online Certificate Status Protocol (OCSP) extension, .NET Core on OSX will fail to connect due to a limitation of the Apple Security Framework (see https://github.com/dotnet/corefx/issues/29064). Prior to version 2.0, .NET Core on OSX used OpenSSL, which does support CRLs without OCSP.
+#### Notes
```

Comment:
Rendered preview: ![image](https://user-images.githubusercontent.com/33040432/42242193-0e6a8d2e-7edb-11e8-8620-83d79054c076.png)

Reply:
":" Removed since it looked odd in the navigation menu.

Reply:
Tweaked the format of the bullets as best I could within the limits of Hugo + Markdown